cypress/integration/t3.js

```diff
@@ -0,0 +1,35 @@
+import T3Page from '../obj/t3Page';
+import LoginPage from '../obj/loginPage';
+
+const t3page = new T3Page();
+const loginPage = new LoginPage();
+
+describe('Sourcebooks login', function() {
+
+    it('Admin can create new Task', function(){
+        let taskName = Math.random();
+        let taskDescription = Math.random();
+        let rateValue = Math.floor(Math.random() * 10) + 1;
+
+        loginPage.visit();
+        loginPage.loginAsSpecificUserAndRole("Valentinas Kasteckis", "Admin");
+        loginPage.visitSpecificURL('/tasks');
+
+        t3page.typeInTaskName(taskName);
+        cy.wait(3000); // for some reason JS does not have enough time to load in, so this helps
```

Comment:
I understand what you're saying. It's fine to keep it here I think, but looking at the long run it's quite bad practice to use wait/sleep in your tests because it's not robust at all, and depending on the internet connection the given milliseconds may not be enough. It's usually best to wait for some element to become visible/invisible instead of putting a thread to sleep. In our system I think you could try waiting for the 'loading' message to disappear, but you can leave this 'wait' for the time being.